tests/server.c

```diff
@@ -311,7 +311,7 @@ main(int argc, const char **argv)
       }
 
       result = rustls_web_pki_client_cert_verifier_builder_add_crl(
-        client_cert_verifier_builder, (uint8_t *)crlbuf, certbuf_len);
+        client_cert_verifier_builder, (uint8_t *)crlbuf, crlbuf_len);
```

Comment:
looks like this was my copy/paste error in 10d1be7e6620f311f73b2ff1be94a15b5b40f01b 

Thanks for fixing.